### PR TITLE
Add E2E faucet test

### PR DESCRIPTION
**Pull request by opendansor**
### Description of the Change

Add an E2E test for a user with the faucet command. 

### Quantitative Performance Benefits

Testing to make sure this feature doesn't regress by accident. 

### Possible Drawbacks

None

### Verification Process

The test passes.

### CODE REVIEW

File: tests/e2e_tests/utils.py [hunk 1]
```diff
@@ -3,9 +3,10 @@
 import subprocess
 import sys
 
-import requests
-from substrateinterface import Keypair
 from typing import List
+
+from substrateinterface import Keypair
```
Comment (basfroman) [inline tests/e2e_tests/utils.py]: FYI: we have `Keypair` imported into `bittensor/__init__.py`. You can use it as `bittensor.Keypair`.

**Review by basfroman — APPROVED** [review]
lgtm

File: tests/e2e_tests/utils.py (hunk 1 above)
Comment (opendansor) [reply tests/e2e_tests/utils.py]: Got it. Importing from bittensor

**Comment by opendansor** [comment]
This PR will have test failing until [https://github.com/opentensor/subtensor/pull/515](https://github.com/opentensor/subtensor/pull/515/files) is merged in.

**Review by thewhaleking — APPROVED** [review]

**Review by ibraheem-abe — APPROVED** [review]

**Pull request merged**